[2024-07-30T10:26:29.054+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:26:29.173+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-01T00:00:00+00:00 [queued]>
[2024-07-30T10:26:29.210+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-01T00:00:00+00:00 [queued]>
[2024-07-30T10:26:29.212+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:26:29.341+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-01 00:00:00+00:00
[2024-07-30T10:26:29.387+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=909) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:26:29.401+0000] {standard_task_runner.py:63} INFO - Started process 916 to run task
[2024-07-30T10:26:29.391+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-01T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmp3p41haor']
[2024-07-30T10:26:29.410+0000] {standard_task_runner.py:91} INFO - Job 80: Subtask Run_data_quality_checks
[2024-07-30T10:26:29.690+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-01T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:26:30.265+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-01T00:00:00+00:00'
[2024-07-30T10:26:30.283+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:26:30.321+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:26:30.351+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:30.382+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:26:30.387+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:30.390+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 266 records
[2024-07-30T10:26:30.390+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:26:30.403+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:30.423+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:26:30.684+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:30.701+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 28 records
[2024-07-30T10:26:30.702+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:26:30.720+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:30.747+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:26:30.752+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:30.755+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:26:30.755+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:26:30.768+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:30.783+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:26:30.786+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:30.789+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:26:30.790+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:26:30.803+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:30.822+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:26:30.966+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:30.968+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 266 records
[2024-07-30T10:26:30.969+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:26:30.984+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-01T00:00:00+00:00, execution_date=20181101T000000, start_date=20240730T102629, end_date=20240730T102630
[2024-07-30T10:26:31.103+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:26:31.124+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
